src/mixer/basetrackplayer.cpp

```diff
@@ -415,8 +416,16 @@ void BaseTrackPlayerImpl::slotLoadFailed(TrackPointer pTrack, const QString& rea
         qDebug() << "Failed to load track (NULL track object)" << reason;
     }
     m_pChannelToCloneFrom = nullptr;
+
     // Alert user.
+    // Show only one message for this track at a time
+    if (pTrack && m_pPrevFailedTrack == pTrack) {
+        return;
+    } else if (pTrack) {
+        m_pPrevFailedTrack = pTrack;
```

Comment:
Keeping TrackPtr cached for an unforeseeable time is not a good idea. Use the id.

Reply:
Makes sense, thanks.
Should I initialise (in the constructor) with `m_pPrevFailedTrack()` or `m_pPrevFailedTrack(TrackId)`?

Reply:
Sorry, @ronso0, I am no longer available for reviews. This was just a hint to prevent a regression.